recipes/fox/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "fox" %}
+{% set version = "4.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/andreww/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: dea0adc9cc035238fa9cdba42f2bf56481e3a64ac8aa0aece9119f127f71d4e7
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - m4
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libFoX_fsys.a
+    - test -f ${PREFIX}/lib/libFoX_utils.a
+    - test -f ${PREFIX}/lib/libFoX_common.a
+    - test -f ${PREFIX}/lib/libFoX_wxml.a
+    - test -f ${PREFIX}/lib/libFoX_wkml.a
+    - test -f ${PREFIX}/lib/libFoX_sax.a
+    - test -f ${PREFIX}/lib/libFoX_dom.a
+
+about:
+  home: http://homepages.see.leeds.ac.uk/~earawa/FoX/
+  license: OTHER
```

It's a mix of Zlib  and BSD-3-Clause I think, in which case see: https://spdx.org/ids-how
```suggestion
  license: Zlib AND BSD-3-Clause
```